server/src/internalClusterTest/java/org/elasticsearch/cluster/settings/ClusterSettingsUpdateWithFaultyMasterIT.java

```diff
@@ -62,7 +62,7 @@ public void testClusterSettingsUpdateNotAcknowledged() throws Exception {
             .cluster()
             .prepareUpdateSettings()
             .setPersistentSettings(Settings.builder().put(BlockingClusterSettingTestPlugin.TEST_BLOCKING_SETTING.getKey(), true).build())
-            .setMasterNodeTimeout(TimeValue.timeValueMillis(10L))
+            .setMasterNodeTimeout(TimeValue.timeValueMillis(100L))
```

Comment:
This does not make the test run longer.
My assumption is that this timeout is only needed to prevent test hanging for extended period of time in case of test failure.